circle.yml

```diff
@@ -7,10 +7,11 @@ machine:
   services:
     - redis
   php:
-    version: 7.0.4
+    version: 7.0.7
 
 dependencies:
   pre:
+    - echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" | mysql
```

Comment:
which issue lead you to do this ?

Reply:
This was necessary to avoid errors caused by pagerFanta's queries due to CI's sql conf. Approved by Baptiste Fourcher !